src/ldp.rs

```diff
@@ -227,6 +252,14 @@ impl LinkedDataProofs {
                 match &curve[..] {
                     "secp256k1" => {
                         if algorithm.as_ref() == Some(&Algorithm::ES256KR) {
+                            if use_epsig(options, key) {
+                                #[cfg(feature = "keccak-hash")]
+                                return EthereumPersonalSignature2021
+                                    .sign(document, options, &key)
+                                    .await;
+                                #[cfg(not(feature = "keccak-hash"))]
+                                return Err(Error::ProofTypeNotImplemented);
```

Comment:
Wondering if we should have another error for "feature not enabled" or something like that. I don't think it would massively help someone debug though

Reply:
Started adding that in #218: https://github.com/spruceid/ssi/pull/218/commits/23e9018fc25a5bddbc3a26888aacfef9774acd36#diff-97e25e2a0e41c578875856e97b659be2719a65227c104b992e3144efa000c35e